modules/alloy/src/main/ts/ephox/alloy/api/ui/GuiTypes.ts

```diff
@@ -7,19 +7,11 @@ import * as FunctionAnnotator from '../../debugging/FunctionAnnotator';
 
 const premadeTag = Id.generate('alloy-premade');
 
-const premade = (comp: AlloyComponent): PremadeSpec => {
-  return Objects.wrap(premadeTag, comp);
-};
+const premade = (comp: AlloyComponent): PremadeSpec => Objects.wrap(premadeTag, comp);
 
-const getPremade = (spec: AlloySpec): Option<AlloyComponent> => {
-  return Obj.get<any, string>(spec, premadeTag);
-};
+const getPremade = (spec: AlloySpec): Option<AlloyComponent> => Obj.get<any, string>(spec, premadeTag);
 
-const makeApi = <A, R>(f: (api: A, comp: AlloyComponent, ...rest: any[]) => R) => {
-  return FunctionAnnotator.markAsSketchApi((component: AlloyComponent, ...rest: any[]/* , ... */) => {
-    return f.apply(undefined, ([ component.getApis<any>() ] as any).concat([ component ].concat(rest)));
-  }, f);
-};
+const makeApi = <A, R>(f: (api: A, comp: AlloyComponent, ...rest: any[]) => R) => FunctionAnnotator.markAsSketchApi((component: AlloyComponent, ...rest: any[]/* , ... */) => f.apply(undefined, ([ component.getApis<any>() ] as any).concat([ component ].concat(rest))), f);
```

Comment:
Another one that's hard to read. I think this and the last one just needs to be put on new lines, instead of being on a single line?

Reply:
Fixed